screens/FindGamePage.js

```diff
@@ -0,0 +1,251 @@
+        <View style = {{ paddingTop: '7.5%' }}>
+          <View style = { styles.time }>
+            <View style = { styles.timeTop }>
+              <Image style = {{ height: 40, width: 40 }} source={require('./../assets/time.png')}/>
+              <Text style = { this.state.fontLoaded ? styles.headerText : styles.headerTextElse }>Time</Text>
+            </View>
+            <View style = {{ height: 3, width: '100%', backgroundColor: '#fff' }}>
+            </View>
+            <View style = { styles.timeBottom }>
+              <View style = { styles.optionButtons }>
+                <Text style = { this.state.fontLoaded ? styles.optionsTimeText : styles.optionsTimeTextElse }>Now</Text>
+              </View>
+              <View style = { styles.optionButtons }>
+                <Text style = { this.state.fontLoaded ? styles.optionsTimeText : styles.optionsTimeTextElse }>Later</Text>
+              </View>
+            </View>
+          </View>
+        </View>
+        <View style = {{ paddingTop: '3%' }}>
+          <View style = { styles.place }>
+            <View style = { styles.placeTop }>
+              <Image style = {{ height: 40, width: 40 }} source = { require('./../assets/place.png') }/>
+              <Text style = { this.state.fontLoaded ? styles.headerText : styles.headerTextElse }>Place</Text>
+            </View>
+            <View style = {{ height: 5, width: '100%', backgroundColor: '#fff' }}>
+            </View>
+            <View style = { styles.placeBottom }>
+              <FlatList
+                data = {[
+                  { key: 'Alpha Phi Alpha' },
+                  { key: 'Alpha Chi' },
+                  { key: 'Alpha Theta' },
+                  { key: 'Alpha Phi' },
+                  { key: 'Alpha Pi Omega' },
+                  { key: 'AXiD' },
+                  { key: 'Beta' },
+                  { key: 'BG' },
+                  { key: 'Chi Delt' },
+                  { key: 'Chi Gam' },
+                  { key: 'EKT' },
+                  { key: 'GDX' },
+                  { key: 'Heorot' },
+                  { key: 'Kappa' },
+                  { key: 'KD' },
+                  { key: 'KDE' },
+                  { key: 'Lambda Upsilon Lambda' },
+                  { key: 'Phi Delt' },
+                  { key: 'Phi Tau' },
+                  { key: 'Psi U' },
+                  { key: 'Sig Ep' },
+                  { key: 'Sig Nu' },
+                  { key: 'Sigma Delt' },
+                  { key: 'Sigma Lambda Upsilon' },
+                  { key: 'Tabard' },
+                  { key: 'TDX' },
+                  { key: 'Zete' },
+                ]}
+                renderItem = {({ item }) => (
+                  <View style = {{ paddingLeft: 5, paddingTop: 2, paddingBottom: 2 }}>
+                    <View style = { styles.optionButtons }>
+                      <Text style = { this.state.fontLoaded ? styles.optionsTextPlace : styles.optionsTextPlaceElse }>{ item.key }</Text>
```

Comment:
`styles.optionsTextPlaceElse` isn't defined anywhere. Either remove it or defined it